### ISSUE TITLE
Electrum + Ledger Nano mnemonic

### ISSUE THREAD

**Issue by rem0g** [gurnec/btcrecover#57]
Hello, 

I don't know if my question is well fit here. I have wrote down mnemonic in a random order I don't remember exactly anymore, after a few attempts with seedrecover.py there is no any luck. It did 12 million tries, which looks not alot for me. I've done this before with my script and came up to 100 million tries.

After that I've generated a new set of mnemonic (24 words), extracted xpub key from that in Electrum and randomized the seed in 1-12-2-13-3-14-etc order. With xpub-match and the randomized seed there is also no recover. 

I've tested the correct order and it worked well.

So in short: Randomized seed with xpub from electrum cannot be recovered.

Question: How do I explain to seedrecover.py it has to try all attempts? Does it need a correct checksum at #3 and #24 to work well? 

Thanks in advance.

**Comment by gurnec** [comment]
How many words long is the mnemonic you're looking for?

If it's 18 or 24 words long, there are unfortunately far too many permutations to fully brute-force without knowing a lot more about the order (for 18, over 6 quadrillion, i.e. 6 billion million). If it's 12 words long, it may be possible (479 million permutations).

**Comment by rem0g** [comment]
Thank you for your kind reply. I know the first eight words in correct order, so that would leave me with 16 words I have to put in correct order. 

How do I instruct seedrecover.py to try all premutations between 8 - 24 words or if guessed correctly 12-24 words? I've adjusted the code and passed a batch-file with some guesses to the seedrecover.py but no luck so far. 

I'm happy if there is any workable solution I can try to code something.

**Comment by gurnec** [comment]
Just to make sure I understand, for words in positions 1 - 8 you are **positive** you are right. For words 9 - 12 you _might_ be right but _not_ positive. And words 13 - 24 you have no idea. Is that right?

If you're right about words 1 - 12, then that's not too bad (479 million permutations like I said earlier). If you're _only_ right about words 1 - 8, then it's about 21 trillion combinations, which is probably too many....

In any case, I don't think seedrecover can currently do what you want. It's not impossible to make work, but I'll have to think about it when I have some spare time....

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover
